app/styles/ui/_tutorial.scss

```diff
@@ -1,12 +1,117 @@
 .tutorial-panel-component {
+  display: flex;
+  flex-flow: column nowrap;
   min-width: 300px;
+  max-width: 350px;
+  background-color: var(--background-color);
+  box-shadow: var(--base-box-shadow);
+  z-index: var(--side-panel-z-index);
```

Comment:
Is there any scenario where this panel will actually sit on top of anything? Could we live without defining an explicit z-index?

Reply:
unfortunately we do for the panel's shadow to appear on top of the codemirror buffer 😞

i'll add a comment to that effect